controllers/comments.controllers.js

```diff
@@ -1,32 +1,36 @@
-const selectArticleById  = require("../models/articles.models");
+const{ selectArticleById } = require("../models/articles.models");
 const { insertCommentsByArticleId, selectCommentsByArticleId } = require("../models/comments.models");
 
 exports.getCommentsByArticleId = (req, res, next) => {
-    const { article_id } = req.params;
-    selectArticleById(article_id) 
-      .then(article => {
-        if (!article) {
+  const { article_id } = req.params;
+  Promise.all([
+      selectArticleById(article_id),
+      selectCommentsByArticleId(article_id)
+  ])
+  .then(([article, comments]) => {
+      if (!article) {
```

Comment:
This error should already be thrown by selectArticleById, we don't need to throw the error here